### PR TITLE
refactor: converted queries in controllers

### PR DESCRIPTION
**Pull request by Aradhya-Tripathi**
Moving from raw queries to ```frappe.qb```

### CODE REVIEW

File: healthcare/controllers/queries.py [hunk 1]
```diff
@@ -4,14 +4,10 @@
...
-		from `tabHealthcare Service Unit`
-		where
-			is_group = 0
-			and company = {company}
-			and name like {txt}""".format(
-				company = frappe.db.escape(filters.get('company')), txt = frappe.db.escape('%{0}%'.format(txt)))
+	table = frappe.qb.DocType("Healthcare Service Unit")
+	query = frappe.qb.from_(table).where(table.is_group == 0) \
+	.where(table.company == frappe.db.escape(filters.get('company'))) \
+    .where(table.name.like(frappe.db.escape('%{0}%'.format(txt)))).get_sql()
```
Comment (Aradhya-Tripathi) [inline healthcare/controllers/queries.py:10]: @gavindsouza  will `frappe.db.escape` cause a similar problem?
ref -> https://github.com/frappe/frappe/issues/15087

Comment (gavindsouza) [reply healthcare/controllers/queries.py:10]: All user inputs should be escaped, but they weren't here before. @ChillarAnand Any reason for this?

---

> will frappe.db.escape cause a similar problem?

If you can't anticipate it, DIY. Nothing better than good ol' trying it out yourself :trollface:

Comment (Aradhya-Tripathi) [reply healthcare/controllers/queries.py:10]: So doesn't seem like a problem

**Review by ChillarAnand — APPROVED** [review]

**Comment by gavindsouza** [comment]
@Aradhya-Tripathi we don't need `frappe.db.escape` since parameterization though, do we?

**Pull request merged**